src/acrn/acrn_driver.c

```diff
@@ -126,6 +126,17 @@ acrnIsRtvm(virDomainDefPtr def)
     return (nsdef && nsdef->rtvm);
 }
 
+static char*
+acrnGetCpuAffinity(virDomainDefPtr def)
+{
+    acrnDomainXmlNsDefPtr nsdef = def->namespaceData;
+
+    if (nsdef)
+	return nsdef->cpu_affinity;
```

Comment:
Looks like the indentation is off on this line?

Reply:
Updated